### ISSUE TITLE
Update 1.20.6

### ISSUE THREAD

**Issue by Triskelen** [itzg/docker-minecraft-server#2820]
### Describe the problem

Since updating the package to 1.20.6 my server no longer alowes remote connections. 1.20.5 workt fine but now nobody can remotely conectect to my minecraft server. No router or switch parameters where changed or did the minecraft port change?

### Container definition

```
Paste run command or compose file here
```

### Container logs

```
Paste logs here
```

**Comment by itzg** [comment]
Please provide container config and logs.

**Comment by m4tta** [comment]
```
text  error  warn  system  array  login  

[mc-image-helper] 09:16:23.511 ERROR : Invalid parameter provided for 'resolve-minecraft-version' command: Minecraft version ' 1.20.6' is not valid
[mc-image-helper] 09:16:59.674 ERROR : Invalid parameter provided for 'resolve-minecraft-version' command: Minecraft version ' 1.20.6' is not valid
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 1 1000 1000 416 May 12 01:31 /data'
[init] Resolving type given vanilla
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 1 1000 1000 406 May 12 09:16 /data'
[init] Resolving type given vanilla

** Press ANY KEY to close this window ** 
```

**Comment by Shingoo2012** [comment]
Greetings, 

I seem to also be having a similar issue with the new version. 

LOG:
2024/05/12 15:26:26 | stdout | [init] Setting initial memory to 1G and max to 1G
2024/05/12 15:26:10 | stdout | [init] Resolved version given LATEST into 1.20.6
2024/05/12 15:26:05 | stdout | [init] Resolving type given VANILLA
2024/05/12 15:26:05 | stdout | [init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 1 1000 1000 592 May 12 13:25 /data'
2024/05/12 15:25:59 | stdout | 2024-05-12T13:25:59.690Z INFO mc-server-runner Done
2024/05/12 15:25:59 | stdout | 2024-05-12T13:25:59.689Z WARN mc-server-runner Minecraft server failed. Inspect logs above for errors that indicate cause. DO NOT report this line as an error. {"exitCode": 1}
2024/05/12 15:25:59 | stdout | java.lang.UnsupportedClassVersionError: net/minecraft/bundler/Main has been compiled by a more recent version of the Java Runtime (class file version 65.0), this version of the Java Runtime only recognizes class file versions up to 61.0
2024/05/12 15:25:59 | stdout | Error: LinkageError occurred while loading main class net.minecraft.bundler.Main
2024/05/12 15:25:59 | stdout | [init] Starting the Minecraft server...

Server Settings:
{
   "CapAdd" : null,
   "CapDrop" : null,
   "cmd" : "",
   "cpu_priority" : 90,
   "enable_publish_all_ports" : false,
   "enable_restart_policy" : true,
   "enable_service_portal" : null,
   "enabled" : false,
   "env_variables" : [
      {
         "key" : "PATH",
         "value" : "/opt/java/openjdk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
      },
      {
         "key" : "JAVA_HOME",
         "value" : "/opt/java/openjdk"
      },
      {
         "key" : "LANG",
         "value" : "en_US.UTF-8"
      },
      {
         "key" : "LANGUAGE",
         "value" : "en_US:en"
      },
      {
         "key" : "LC_ALL",
         "value" : "en_US.UTF-8"
      },
      {
         "key" : "JAVA_VERSION",
         "value" : "jdk-17.0.8+7"
      },
      {
         "key" : "TYPE",
         "value" : "VANILLA"
      },
      {
         "key" : "VERSION",
         "value" : "LATEST"
      },
      {
         "key" : "EULA",
         "value" : "TRUE"
      },
      {
         "key" : "UID",
         "value" : "1000"
      },
      {
         "key" : "GID",
         "value" : "1000"
      }
   ],
   "exporting" : false,
   "id" : "dfe2e13902fd5be6abca4d10c7c65e918d918abc2c0ab7d769bb19f0bbc4627b",
   "image" : "itzg/minecraft-server:latest",
   "is_ddsm" : false,
   "is_package" : false,
   "labels" : {
      "org.opencontainers.image.authors" : "Geoff Bourne <itzgeoff@gmail.com>",
      "org.opencontainers.image.created" : "2023-08-23T02:35:07.044Z",
      "org.opencontainers.image.description" : "Docker image that provides a Minecraft Server that will automatically download selected version at startup",
      "org.opencontainers.image.licenses" : "Apache-2.0",
      "org.opencontainers.image.ref.name" : "ubuntu",
      "org.opencontainers.image.revision" : "8fd149579526532c120d173f9864918bf645abb6",
      "org.opencontainers.image.source" : "https://github.com/itzg/docker-minecraft-server",
      "org.opencontainers.image.title" : "docker-minecraft-server",
      "org.opencontainers.image.url" : "https://github.com/itzg/docker-minecraft-server",
      "org.opencontainers.image.version" : "java17"
   },
   "links" : [],
   "memory_limit" : 4395630592,
   "name" : "itzg-minecraft-server2",
   "network" : [
      {
         "driver" : "bridge",
         "name" : "bridge"
      }
   ],
   "network_mode" : "bridge",
   "port_bindings" : [
      {
         "container_port" : 25565,
         "host_port" : 25565,
         "type" : "tcp"
      },
      {
         "container_port" : 25565,
         "host_port" : 25565,
         "type" : "udp"
      }
   ],
   "privileged" : false,
   "shortcut" : {
      "enable_shortcut" : false,
      "enable_status_page" : false,
      "enable_web_page" : false,
      "web_page_url" : ""
   },
   "use_host_network" : false,
   "version" : 2,
   "volume_bindings" : [
      {
         "host_volume_file" : "/docker/MineCraft",
         "is_directory" : true,
         "mount_point" : "/data",
         "type" : "rw"
      }
   ]
}

**Comment by itzg** [comment]
Re-pull the image or explicitly use the `itzg/minecraft-server:java21` tagged image:

https://docker-minecraft-server.readthedocs.io/en/latest/versions/java/

**Comment by Shingoo2012** [comment]
> Re-pull the image or explicitly use the `itzg/minecraft-server:java21` tagged image:
> 
> https://docker-minecraft-server.readthedocs.io/en/latest/versions/java/

Newbie question, how do I actually re-pull the image? I'm using Container on a Synology NAS. I'm guessing I would run a console command, I do not see anywhere where I can edit specifically the image parameters within Container that would force it to Re-pull the image.

**Comment by itzg** [comment]
I'm not familiar with Synology NAS. You'll need to consult their docs or contact them.

**Comment by Shingoo2012** [comment]
> ...or just set the image tag, as I mentioned.

Thank you I was able to get the server to work again!

**Comment by Shingoo2012** [comment]
wondering if anyone is having issues with the server have issues setting the new seed.

I sed my seed to 1234abc, the server loaded but when I went into the console to see what seed showed, it listed as 1511032834.

wondering if I'm doing something wrong, maybe a known issue?

**Comment by itzg** [comment]
Text seeds always get converted into the numerical equivalent, so I'm not sure if you're seeing an actual issue.

**Comment by Shingoo2012** [comment]
> Text seeds always get converted into the numerical equivalent, so I'm not sure if you're seeing an actual issue.

Okay admittedly, my "issue" is that I'm learning some aspects of Minecraft I never knew before. In big picture I'm trying to find a new world to play with my kids. We have been using https://www.chunkbase.com/ (Java 1.20) to find a new seed. We enter in seeds like, 123abcd, JesusLovesMe2012, into chunkbase, and found the coordinates of an area we would like to use as our starting location. 

When I put these seeds into the server config file, we boot up into a new world, I then transport to those coordinates we found on chunkbase, but its a totally different area than what chunk base generated. Also looking up the seed in the console and got a numeric seed instead of the alpha-numeric we entered in the file (makes sense based on what you said). BUT, in reverse, if I put the numeric seed value the server says I'm using into ChunkBase, I get a different world.

In the big picture, how can I properly use the Chunkbase page to "shop" around for worlds that I know will translate properly into the minecraft server?

I apologize as this discussion has evolved into a topic that probably belongs somewhere else.
